plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/transactionlog/checkpoint/TestTransactionLogTail.java

```diff
@@ -66,8 +63,7 @@ private List<DeltaLakeTransactionLogEntry> readJsonTransactionLogTails(String ta
             throws Exception
     {
         TrinoFileSystem fileSystem = new HdfsFileSystemFactory(HDFS_ENVIRONMENT).create(SESSION);
-        URI resource = getClass().getClassLoader().getResource(tableLocation).toURI();
-        Path tablePath = new Path(resource);
+        String tablePath = getClass().getClassLoader().getResource(tableLocation).toURI().toString();
```

Comment:
Based on the latest changes from ebe2554a the term `path` has been changed to `location`.